### DISCUSSION TITLE
June 5 Stands-up

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#210] [Q&A] [answered]
@hollowaykeanho , please update. Do expect Jean's incoming. =)

**Comment by JeanShuralyov** [comment]
Oh no release for last week? Ouch.

**Comment by hollowaykeanho** [comment, marked as answer]
Yeah... Estimate on safe side needs 2 weeks. I'm currently working on the last porting section: `specs`. 

No signs of performance degradation anymore and everything works one quite well (in fact, double the loads as I'm translating to simplified Mandarins as well).

`specs` section is a complete overhaul because of the copyright countermeasure and also the packaging renaming.

**Comment by corygalyna** [comment]
I see. Project underestimated. We need to hard it.. it's June so this month must have a hard release.

**Comment by hollowaykeanho** [comment]
Within this month then yes. It's entirely possible. I didn't add any new feature. All the current efforts are just documentation porting.

**Comment by JeanShuralyov** [comment]
Nice!!!

**Comment by corygalyna** [comment]
Ok. We're good then. Please complete it.
